File: src/main/java/org/polyfrost/animatium_legacy/hooks/PatcherConfigHook.java
package org.polyfrost.animatium_legacy.hooks;

import club.sk1er.patcher.config.PatcherConfig;
import org.polyfrost.animatium_legacy.Animatium;

public final class PatcherConfigHook {
    public static boolean isParallaxFixEnabled() {
        return Animatium.isPatcherPresent && PatcherConfig.parallaxFix;
    }
}

File: src/main/kotlin/org/polyfrost/animatium_legacy/hooks/PatcherConfigHook.kt
package org.polyfrost.animatium_legacy.hooks

import club.sk1er.patcher.config.PatcherConfig
import org.polyfrost.animatium_legacy.Animatium

object PatcherConfigHook {
    @JvmStatic
    fun isParallaxFixEnabled(): Boolean = Animatium.isPatcherPresent && PatcherConfig.parallaxFix
}